modules/enabling-habana-gaudi-devices.adoc

```diff
@@ -0,0 +1,56 @@
+:_module-type: PROCEDURE
+
+[id='enabling-habana-gaudi-devices_{context}']
+= Enabling Habana Gaudi devices
+
+[role='_abstract']
+Before you can use Habana Gaudi devices in {productname-short}, you install the necessary dependencies and deploy the HabanaAI Operator in your deployment. 
+
+.Prerequisites
+* You have logged in to {openshift-platform}.
+* You have the `cluster-admin` role in {openshift-platform}.
+
+.Procedure
+. Configure an Amazon EC2 DL1 instance link:https://docs.habana.ai/en/latest/AWS_EC2_DL1_and_PyTorch_Quick_Start/AWS_EC2_DL1_and_PyTorch_Quick_Start.html[AWS DL1 Quick Start Guide].
+. To enable Habana Gaudi devices in {productname-short}, follow the instructions at link:https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/index.html[HabanaAI Operator for OpenShift].
+. Create an accelerator profile for your Habana Gaudi device. 
+.. In the {openshift-platform} web console in the *Administrator* perspective, click *Administration* -> *CustomResourceDefinitions*.
+.. Click the `acceleratorprofiles.dashboard.opendatahub.io` custom resource definition (CRD).
++
+A details page for the custom resource definition (CRD) opens.
+.. Click the *Instances* tab.
+.. Click *Create AcceleratorProfile*. 
++ 
+The *Create AcceleratorProfile* page opens and displays an embedded YAML editor.
+.. Enter or paste the YAML code directly in the embedded editor. For example:
```

Comment:
Same question as above RE: YAML code coming from here? 

Aware of advising 'to paste the YAML code' if we've not referenced copying specific code elsewhere.